scapy/layers/inet.py

```diff
@@ -1382,9 +1382,7 @@ def world_trace(self):
         # Return the drawn lines
         return lines
 
-    def make_graph(self, ASres=None, padding=0):
-        if ASres is None:
-            ASres = conf.AS_resolver
+    def make_graph(self, ASres=None, padding=0): 
```

Comment:
The trailing space char breaks the flake8 tests.

Reply:
Fixed